### DISCUSSION TITLE
How to connect existing installation of Python to AEDT

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#4097] [Q&A] [answered]
Hi,

Is there a document that shows step-by-step how to setup/connect an existing installation of Python (for example, anaconda/mini-conda) to Ansys HFSS for PyAEDT scripting?

Thanks in advance.

**Comment by z7766918** [comment]
Hi,
You can refer to this website, which is the document of PyAEDT.
https://aedt.docs.pyansys.com/version/stable/

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Abhi-SimuTech ,

As @z7766918 suggested, the info is in the documentation. In the installation page, you can find different ways of installing PyAEDT:
https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html


If I am not wrong, what you are trying is to create a PyAEDT "launcher" from AEDT. In this case, from 2023R2 is possible, and it was this script is doing:
https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#install-from-a-python-file

In this case, the script is creating a new virtual environment using the CPython base intepreter of AEDT (this new venv is called pyaedt_venv_IDE, then installing PyAEDT, and finally creating a new button in AEDT.

YOu need to customize this script, and instead of creating a new virtual environment, you just need to point to the existing one with PyAEDT.  What you ask is something very specific, but if you think it is useful for the community, please share the modified script and we can discuss if this could be added to the documentation.
